### PR TITLE
fix: exclude `bookmarkName` from `advancedSearchParams`

### PR DESCRIPTION
**Pull request by jamil314**

### CODE REVIEW

File: packages/client/src/views/AdvancedSearch/SaveBookmarkModal.tsx [hunk 1]
```diff
@@ -59,11 +59,13 @@ export function BookmarkAdvancedSearchModal({
   const { searchId, ...advancedSearchParams } = useSelector(
     getAdvancedSearchParamsState
   )
+  delete advancedSearchParams.bookmarkName
```
Comment (Zangetsu101) [inline packages/client/src/views/AdvancedSearch/SaveBookmarkModal.tsx:62]: Why is it that we need to remove the `bookmarkName`?

Comment (jamil314) [reply packages/client/src/views/AdvancedSearch/SaveBookmarkModal.tsx:62]: gateway type [AdvancedSeachParameters](https://github.com/opencrvs/opencrvs-core/blob/5986f91185cdd9e9dc05f219b62dee4d2473f521/packages/client/src/utils/gateway.ts#L105-L106) does not have `bookmarkName`

Comment (Zangetsu101) [reply packages/client/src/views/AdvancedSearch/SaveBookmarkModal.tsx:62]: You mean [AdvancedSeachParametersInput](https://github.com/opencrvs/opencrvs-core/blob/5986f91185cdd9e9dc05f219b62dee4d2473f521/packages/client/src/utils/gateway.ts#L166) instead?

File: packages/client/src/views/AdvancedSearch/SaveBookmarkModal.tsx [hunk 2]
```diff
@@ -59,11 +59,13 @@ export function BookmarkAdvancedSearchModal({
   const { searchId, ...advancedSearchParams } = useSelector(
```
Comment (Zangetsu101) [inline packages/client/src/views/AdvancedSearch/SaveBookmarkModal.tsx:59]:
```suggestion
  const { searchId, bookmarkName, ...advancedSearchParams } = useSelector(
```

**Review by Zangetsu101 — APPROVED** [review]

**Pull request merged**